packages/segment/src/integration.ts

```diff
@@ -0,0 +1,17 @@
+import { SegmentNotifierAdapter } from './notifier-adapter';
+import { SegmentEventListenerAdapter } from './event-listener-adapter';
+
+export interface Config {
+    eventListenerAdapter: {
+        parametersToReadFromSegment: Record<string, string>
+    },
+    maxWaitTime?: number;
+}
+
+export function integration(config: Config): void {
+    const segmentNotifier = new SegmentNotifierAdapter(config.maxWaitTime);
```

Comment:
I had to assign this to a var (and line 16) otherwise SonarCloud said its a major bug and fails